index.html

```diff
@@ -38,6 +39,204 @@ <h2 id="job-title">I am a Software Developer</h2>
+                </div>
+                <div class="last-project project-1">
+                    <a id="project-backdrop" href=""><img src="images/Img Placeholder-2.png" alt=""></a>
+                    <div class="floating">
+                        <div>
+                            <h3 id="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p id="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        </div>
+                        <div>
+                            <a href=""><button class="project-1-buttons">html</button></a>
+                            <a href=""><button class="project-1-buttons" >bootstrap</button></a>
+                            <ahref=""><button class="project-1-buttons">Ruby</button></a>
+                        </div>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button id="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+            </section>
+            <section class="about-me">
+                <div id="about-div">
+                    <h2 id="about-me-title">About Me</h2>
+                </div>
+                <div id="about-me-description-div">
+                    <p id="about-me-description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href=""><button id="get-resume-button">Get My Resume</button></a>
+                </div>
+                <p id="line3"></p>
+                <div class="languages-div">
+                    <div class="languages">
+                        <ul id="languages">Language
+                            <li id="individual-languages">JavaSript</li>
+                            <li id="individual-languages">Ruby</li>
+                            <li id="individual-languages">Html</li>
+                            <li id="individual-languages">Css</li>
+                        </ul>
+                    </div>
+                    <div class="frameworks-div">
+                        <ul id="languages">Frameworks
+                            <div class="mini-language-div">
+                                <li id="individual-frameworks">Bootstrap</li>
+                                <li id="individual-frameworks">Ruby on Rails</li>
+                                <li id="individual-frameworks">RSPec</li>
+                                <li  id="individual-frameworks">CapyBara</li>
+                                <li id="individual-frameworks">Selenium</li>
+                            </div>
+                            
+                        </ul>
```

Comment:
Kindly, I suggest that when you use UL/LI listings, just put LI inside a UL (remove the div that you have inside)